timescaledb/how-to-guides/hypertables/hypertables-and-unique-indexes.md

```diff
@@ -0,0 +1,140 @@
+# Hypertable partitioning and unique indexes
+Hypertable partitioning introduces some restrictions on unique indexes.
+
+## Partitioning columns
+To determine what unique indexes you can create on a hypertable, first
+identify your partitioning columns.
+
+In TimescaleDB, the following columns are partitioning columns:
+*   The time column used to create the hypertable. Every TimescaleDB hypertable
+    is partitioned by time.
+*   Any space-partitioning columns. Space partitions are optional and not
+    included in every hypertable. You have a space-partitioning column if you
+    specify a `partitioning_column` parameter when you call
+    [`create_hypertable`][create_hypertable] or
+    [`create_distributed_hypertable`][create_distributed_hypertable].
+
+## Partitioning and unique indexes
+When you create a unique index on a hypertable, it must contain all the
+partitioning columns of the hypertable. It may contain other columns as well,
+and they may be arranged in any order.
+
+This restriction is necessary to guarantee global uniqueness in the index.
+<!--TODO: PR for link destination to be merged
+For more information, see the section on [indexes in hypertables][local-indexes].
+-->
+
+If you don't include all the partitioning columns, you get this error:
+```
+ ERROR: cannot create a unique index without the column "<COLUMN_NAME>" (used in
+partitioning) 
+```
+
+<highlight type="note">
+In PostgreSQL, a primary key is a unique index with a `NOT NULL` constraint.
+Any information about unique indexes also applies to primary keys.
+</highlight>
+
+### Create unique indexes on a hypertable
+Create a unique index as you normally would in PostgreSQL, using `CREATE UNIQUE
+INDEX`. Make sure to include all partitioning columns in the index.
+
+For example, say you create a hypertable partitioned on `time` and
```

Comment:
```suggestion
For example, to create a hypertable partitioned on `time` and
```

Reply:
We're not demonstrating this here, we're doing some setup, I think "say you" better conveys what's happening.